src/main/external-resources/renderers/Google-ChromecastUltra.conf

```diff
@@ -41,12 +41,5 @@ Supported = f:m4a            n:2      a:aac-lc|he-aac|mp3   m:audio/mp4
 Supported = f:wav            n:2      a:lpcm                m:audio/x-wav
 Supported = f:weba           n:2      a:opus|vorbis         m:audio/webm
 
-# Supported image formats:
```

Comment:
@Sami32 I find that strange, but I guess it depends on *how* they use it. It is for example possible to use BubbleUPnP to play DLNA content on a Chromecast. Since the Chromecast doesn't have a user interface of its own, I'm not entirely sure how people actually "use" UMS and Chromecast.